src/maintainer/adding_pkgs.rst

```diff
@@ -28,8 +28,19 @@ There are multiple ways to get started:
    start by using the `conda-forge helper script for R recipes <https://github.com/bgruening/conda_r_skeleton_helper>`_ instead.
    Then if necessary, you can make manual edits to the recipe.
 #. If it is a python package, you can generate the recipe as a starting point with ``grayskull``.
-   Use ``conda install -c conda-forge grayskull`` to install ``grayskull``, followed by ``grayskull pypi your_package_name`` to generate the recipe. Note that you do *not* necessarily have to use ``grayskull``, and the
-   recipes produced by ``grayskull`` might need to be reviewed and edited. Read more about ``grayskull`` and how to use it `here <https://github.com/conda-incubator/grayskull#introduction>`__.
+
+.. note::
+
+  `Grayskull <https://github.com/conda-incubator/grayskull>`_ is an automatic conda recipe generator. The goal of this project is to generate concise recipes
+  for conda-forge and eventually replace conda skeleton. Presently, Grayskull can generate recipes for Python packages available on PyPI and also those not published on PyPI and are only available as GitHub repositories.
+
+  Installation and usage of ``grayskull``:
+
+  - Run ``conda install -c conda-forge grayskull`` to install ``grayskull``.
+  - Followed by ``grayskull pypi --strict-conda-forge your_package_name`` to generate the recipe.
```

Comment:
I don't think we have a standard for how command placeholders are written in the conda-forge docs, but I think they should be in all caps, to make them different from the rest of the command and hopefully help people understand that they're supposed to change that part of the command.

You could even add a sentence explaining that they should replace "YOUR_PACKAGE_NAME" with the name of their package.

This would be a change to the instructions to specifically help very new people to the command line/terminal.

Reply:
I think we don‘t (at least I've never seen them :)) Might be good to add that to the docs here as well for future contributors? Some sort of writing guidelines or so.

Reply:
I have made all the changes as suggested by @kathatherine, thank you so much for reviewing! :) 
As for what @BastianZim mentioned, I think that would be a quite useful addition. We would need to define and come up with some more guidelines though before adding this. Let me know what you folks think about this, if this sounds like a good addition, we can open a issue to take this discussion forward.

Reply:
I'm not really involved with the docs so I'd refer this to @kathatherine for approval but I think an issue is a great idea.